frontend/src/app/organization/organization-details/organization-details.component.ts

```diff
@@ -1,7 +1,8 @@
-import { Component, Input } from '@angular/core';
+import {Component, Inject, Input} from '@angular/core';
 
-import { Organization } from '../organization.model';
-import { environment } from '../../../environments/environment';
+import {Organization} from '../organization.model';
+import {environment} from '../../../environments/environment';
+import {Meta} from "@angular/platform-browser";
```

Comment:
There's developed `MetatagsService` to be used here.

That way instead of multiple `this.metaService.updateTag` (+ You forgot about setting title) You can simply execute one `this.metatagsService.setMeta(title, tagsAsObject)`.

Reply:
changed